recipes/libgpuarray/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.6.0" %}
+
+package:
+  name: libgpuarray
+  version: {{ version }}
+
+source:
+  fn: libgpuarray-{{ version }}.tar.gz
+  url: https://github.com/Theano/libgpuarray/archive/v{{ version }}.tar.gz
+  sha256: a58a0624e894475a4955aaea25e82261c69b4d22c8f15ec07041a4ba176d35af
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - toolchain
+    - git
```

Comment:
We needed it for the clone before, but it's probably not required anymore.  I'll remove it and update to 0.6.2.